components/director/tools/tools.go

```diff
@@ -0,0 +1,10 @@
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/maxbrunsfeld/counterfeiter/v6"
+)
+
+// This file imports packages that are used when running go generate, or used
```

Comment:
This is a pattern for locking some external tools that are usually used for code generation to a particular version within a module. This way everyone can use the same counterfeither version. However since this is excluded from the build process via the tools comment, whats missing right now is a makefile step that we can run to set the version of the tool for the module.
Something like 
```
install-tools: download
  @echo Installing tools from tools.go
  @cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
```